### PR TITLE
Enhance Ledgers View

### PR DESCRIPTION
**Pull request by eolivelli**
- describe ledgers using metadata
- change background color
- show number of ledgers and total size
- update the seach results by pressing ENTER on any of the fields
- Move Reload cache button to the first tab in SystemStatus view

### CODE REVIEW

File: web/src/main/bvmui/src/styles/_variables.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $colors: (
-  'primary': #4a4e4d, 
+  'primary': #61b6a1,
```
Comment (aluccaroni) [inline web/src/main/bvmui/src/styles/_variables.scss:2]: This color seems to not fit with the overall UI style. 
Can we choose something darker to better contrast with the white characters? @mino181295  do you have any suggestion?

**Review by aluccaroni — CHANGES_REQUESTED** [review]
This should fix https://github.com/diennea/bookkeeper-visual-manager/issues/37.

Tested the changes on a big deployment with multiple applications, everything looks fine. I only see some not expected behaviour regarging metadata reload: it seems that metadata are no longer loaded at startup and that, sometimes, metadata are refreshed even if I don't use the "RELOAD METADATA FROM ZOOKEEPER" button. 
If that's the case (we have some scheduler that periodically refresh) maybe we should add this information to the "system" tab.

See review for a minor nitpick regarding the color choosed.

**Comment by eolivelli** [comment]
@aluccaroni  this is the current result
ledgers

**Comment by eolivelli** [comment]
Schermata 2020-11-16 alle 15 26 18

last version as discussed offline with @aluccaroni

**Comment by eolivelli** [comment]
it seems that metadata are no longer loaded at startup and that, sometimes, 
this is weird, we have fixed the configuration for the reload at boot

`metadata are refreshed even if I don't use the "RELOAD METADATA FROM ZOOKEEPER" button.`
this is #98 committed yesterday

**Review by aluccaroni — APPROVED** [review]

**Pull request merged**